src/Components/Analyzers/src/build/netstandard2.0/Microsoft.AspNetCore.Components.Analyzers.props

```diff
@@ -1,5 +0,0 @@
-<Project>
-  <PropertyGroup>
-    <DisableImplicitComponentsAnalyzers>true</DisableImplicitComponentsAnalyzers>
```

Comment:
@rynowak I deleted the prop file which caused the SDK to not use the implicit analyzer. Here's what it would look like:

Current release of SDK and current release of package - package wins
Current release of SDK and new release of package - SDK wins
New release of SDK and current release of package -- SDK wins
New release of SDK and new release of package -- SDK wins

The only time you should see the analyzer from the package used are you are not already getting the analyzer via the SDK - 

a) if you are not the right TFM 
b) if you have `DisableImplicitComponentsAnalyzers` set
c) You aren't referencing the Web.SDK

(b) is the semi spooky one. But it's no different than what you would get if you set the property and referenced the analyzer.

Reply:
Ah, I missed that. 

How does part 2 work? The current release of the SDK will end up adding its analyzer, so the code in `_RemoveComponentAnalyzer` removes the one from the package?

Reply:
Yup. The SDK unconditionally adds the analyzer (we aren't changing it). The package decides to remove the one that it would have added to the compilation if it sees the SDK's analyzer already in the list.

Reply:
I'll write up some text to the spooky code